circle.yml

```diff
@@ -53,17 +53,7 @@ jobs:
       - run:
           name: Generate Espresso sanity tests
           command: make sanity-test-example-activities
-      - run:
-          name: Run Firebase instrumentation tests
```

Comment:
@Chaoba , I'm not sure whether we need to delete these Firebase instrumentation tests . Will these be a problem with China usage? I'm thinking no. Perhaps we can keep these tests in the CircleCI file and wait to see if we need to eventually delete it because of China/Google stuff...

Reply:
It seems that we could keep these tests